python/cugraph/jaccard/jaccard_wrapper.pyx

```diff
@@ -34,7 +33,7 @@ cpdef nvJaccard(input_graph):
 
```

Comment:
We should document the newly added parameters

Reply:
Will do.

Reply:
Done.